configs/transfer.py

```diff
@@ -0,0 +1,11 @@
+FEATURE_EXTRACTOR_MAP = {
+  299: 'https://tfhub.dev/google/imagenet/inception_v3/feature_vector/5',
+  224: 'https://tfhub.dev/google/imagenet/mobilenet_v2_100_224/feature_vector/5',
+  192: 'https://tfhub.dev/google/imagenet/mobilenet_v2_100_192/feature_vector/5',
+  128: 'https://tfhub.dev/google/imagenet/mobilenet_v2_100_128/feature_vector/5',
+  96: 'https://tfhub.dev/google/imagenet/mobilenet_v2_100_96/feature_vector/5',
+}
```

Comment:
The 100 parameter here indicates a depth multiplier of 1.00; it's essentially another training speed/test accuracy tradeoff where lower has a similar effect to decreasing image resolution, except it's related to number of channels(?) instead of image size. MobileNet distributions ship with depth multipliers ranging from 0.25 to 1.00, with 1.00 being the slowest/most accurate.

As smaller depth multipliers result in a smaller number of trainable params, they may also allow us to fine tune memory usage.